torch/optim/lr_scheduler.py

```diff
@@ -1234,3 +1234,63 @@ def get_lr(self):
                     group['momentum'] = computed_momentum
 
         return lrs
+    
+class AdaptiveLR(_LRScheduler):
+    """Modify the learning rate of each parameter group,
+    depending on the current & previous loss values.
+
+    If current_loss < previous_loss:
+        lr += a*lr (Increment lr by a * lr)
+    If current loss > previous loss:
+        lr -= b*lr (Decrement lr by b * lr)
+
+    Args:
+        optimizer (Optimizer): Wrapped optimizer.
+        a: (float or list): A multiplicative factor to increment learning rate
+            when the current loss is lower than the previous loss.
+        b: (float or list): A multiplicative factor to decrement learning rate
+            when the current loss is higher than the previous loss.
+
+        The increment/decrement of learning rate is done on
+        each lr for each group in optimizer.param_groups
+    Example:
+        >>> # Assuming optimizer has two groups.
+        >>> a_list = [0.1, 0.2]
+        >>> b_list = [0.5, 0.9]
+        >>> scheduler = LambdaLR(optimizer, a=a_list, b=b_list)
```

Comment:
Should example use the new AdaptiveLR instead of LambdaLR?

Reply:
Yeah my bad;;